addon/server.js

```diff
@@ -154,6 +154,7 @@ export default class Server {
     this.timing = this.timing || config.timing || 400;
     this.namespace = this.namespace || config.namespace || '';
     this.urlPrefix = this.urlPrefix || config.urlPrefix || '';
+    this.trackRequests = this.trackRequests || config.trackRequests || (this.environment === 'test');
```

Comment:
Maybe default to false rather than true in all tests. It's probably set to false originally for some reason.